### ISSUE TITLE
Expose publicly the log files for the list generation

### ISSUE THREAD

**Issue by fmarier** [mozilla-services/shavar-list-creation#28]
The list creation script spits out log files which contain all of the URLs and hashes that made it to the binary lists. This is useful for:

- testers trying to debug list update problems
- interested users wanting to scrutinize our lists

**Comment by fmarier** [comment]
Note: the script runs in Jenkins so we'll need to find a way to get the log output out of there.

**Comment by HarshitSohaney** [comment]
Is this something we want to pursue @artines1 ? With rs, we can use the devtools to look deeper into the problem or contact the SRE team for help. What are your thoughts?

**Comment by artines1** [comment]
This is a good way to debug, so I think this is what we want. However, it's unclear to me how we can get the log file out of the environment where we are running our script. Maybe we can commit the log file of the running result?

**Comment by HarshitSohaney** [comment]
> This is a good way to debug, so I think this is what we want. However, it's unclear to me how we can get the log file out of the environment where we are running our script. Maybe we can commit the log file of the running result?

Yeah we can probably keep the log files public and overwrite them each time theres a change? Not sure how often we use these though
